all.json

```diff
@@ -2,6 +2,8 @@
 	"allow": [],
 	"deny": [
 		"07e96d43-381a-46a3-9c16-6daf97213efc.co",
+		"defichainsync.com",
```

Comment:
Seems like we have this one already based on the CI check.

Reply:
Then how come it isn't blocked?

Reply:
https://github.com/polkadot-js/phishing/blob/master/all.json#L2947